ibis/backends/base/sqlglot/compiler.py

```diff
@@ -59,32 +61,55 @@ def __getitem__(self, key: str) -> sge.Var:
         return sge.Var(this=key)
 
 
+class AnonymousFuncGen:
+    __slots__ = ()
+
+    def __getattr__(self, name: str) -> Callable[..., sge.Anonymous]:
+        return lambda *args: sge.Anonymous(
+            this=name, expressions=list(map(sge.convert, args))
+        )
+
+    def __getitem__(self, key: str) -> Callable[..., sge.Anonymous]:
+        return getattr(self, key)
```

Comment:
This isn't used internally anywhere, it's here for consistency with the other function generators.

Happy to remove it until we need it if that's desirable.

Reply:
I like consistency :)